pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SomaticResultUpload.java

```diff
@@ -124,6 +126,8 @@ public class SomaticResultUpload implements HasDdpInstanceId {
             + "LEFT JOIN ddp_instance as ddp ON (ddp.ddp_instance_id = p.ddp_instance_id) "
             + "WHERE p.ddp_participant_id = ? AND ddp.instance_name= ?)";
 
+    private static final String HARD_DELETE_SOMATIC_DOCUMENT = "delete from somatic_documents where id = ?";
```

Comment:
It looks like the pattern here (and elsewhere in DSM) is to prefix the SQL strings with `SQL_`.